assignments/14BuildDeployStreamlitDashboard.md

```diff
@@ -0,0 +1,86 @@
+# CTD_Assignment_14: Build and Deploy a Streamlit Dashboard
+
+This assignment is to be implemented using **Streamlit**.  
+You will **import a dataset**, **build a dashboard**, **visualize insights**, **showcase data cleaning**, and **deploy your app** to **Streamlit Community Cloud**.
+
+---
+
+## **Task 1: Import a Dataset**
+- Choose a public dataset (CSV, Excel, or through an API).
+- Import the dataset into your Python script using Pandas.
```

Comment:
Is there a standard name for the script file when deploying to streamlit.io? The assignment should give explicit instruction on the file to create.